optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/pages/rotation/RotationConfigurationView.java

```diff
@@ -43,8 +43,6 @@
 
     @PostConstruct
     public void init() {
-        final TenantConfiguration configuration = tenantStore.getCurrentTenant().getConfiguration();
-        weekStart.value = configuration.getWeekStart().toString();
-        period.value = configuration.getTemplateDuration().toString();
+        // TODO: Later
```

Comment:
Can you be more specific on this comment? It's okay to add TODO/FIXMEs but we have to think about the people reading that in the future without context. We now know what "Later" means, but after a while we'll forget and/or other people will have no idea what this is about.

Reply:
It is ``// TODO: Setup databinder to TenantConfiguration/Intialize widget values`